src/SignalR/common/Http.Connections.Common/src/NegotiateProtocol.cs

```diff
@@ -247,5 +338,157 @@ private static AvailableTransport ParseAvailableTransport(JsonTextReader reader)
 
             throw new InvalidDataException("Unexpected end when reading JSON.");
         }
+
+        private static AvailableTransport ParseAvailableTransport(ref Utf8JsonReader reader)
+        {
+            var availableTransport = new AvailableTransport();
+
+            while (CheckRead(ref reader))
+            {
+                switch (reader.TokenType)
+                {
+                    case JsonTokenType.PropertyName:
+                        var memberName = reader.ValueSpan;
+
+                        if (ArrayEqual(memberName, TransportPropertyNameBytes))
```

Comment:
Use span.SequenceEqual()

Reply:
Was wondering if there was something for this...

Reply:
Yep, sequence equal works here. See https://github.com/aspnet/SignalR/pull/2734/files#diff-0e5591272fc4a9be570908d247813246R118